api/routers/edr.py

```diff
@@ -40,14 +39,19 @@
 # We can currently only query data, even if we only need metadata like for this endpoint
 # Maybe it would be better to only query a limited set of data instead of everything (meaning 24 hours)
 async def get_locations(
-    bbox: Annotated[str, Query(example="5.0,52.0,6.0,52.1")]
+    bbox: Annotated[str | None, Query(example="5.0,52.0,6.0,52.1")] = None
```

Comment:
I would prefer to leave this mandatory. The specs are unclear on what it should be.
Forcing people to provide a `bbox` might get the to be have. Otherwise it is to easy to just not provide the parameter, which in most cases will put unnecessary load on the system.

Reply:
It is my understanding that bbox functionality is not yet implemented in geoweb, and geoweb will need to be able to get all locations for now. I will investigate it, and make it mandatory if geoweb can provide the bbox.